### PR TITLE
Feature/amg8833

### PR DESCRIPTION
**Pull request by ctacke**

### CODE REVIEW

File: Source/Meadow.Foundation.Peripherals/Displays.Ascii/Driver/Meadow.Displays.Ascii/Meadow.Displays.Ascii.csproj
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/Displays.Ascii/Driver/Meadow.Displays.Ascii/Meadow.Displays.Ascii.csproj:1]: Needs metadata

File: Source/Meadow.Foundation.Peripherals/Displays.Ascii/Driver/Meadow.Displays.Ascii/AsciiDisplay.cs
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/Displays.Ascii/Driver/Meadow.Displays.Ascii/AsciiDisplay.cs:1]: rename to AsciiConsoleDisplay?

File: Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Cp2112/Driver/Cp2112DigitalInputPort.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+﻿using Meadow.Hardware;
+using System;
+
+namespace Meadow.Foundation.ICs.IOExpanders
+{
+    /// <summary>
+    /// Represents a digital input port implementation for the FT232 bus.
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Cp2112/Driver/Cp2112DigitalInputPort.cs:7]: Update comment

File: Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Cp2112/Driver/Cp2112DigitalOutputPort.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+﻿using Meadow.Hardware;
+using System;
+
+namespace Meadow.Foundation.ICs.IOExpanders
+{
+    /// <summary>
+    /// Digital output port for FT232 devices.
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Cp2112/Driver/Cp2112DigitalOutputPort.cs:7]: Update XML comment

File: Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Cp2112/Driver/ICs.IOExpanders.Cp2112.csproj [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <AssemblyName>Cp2112</AssemblyName>
+    <Company>Wilderness Labs, Inc</Company>
+    <PackageProjectUrl>http://developer.wildernesslabs.co/Meadow/Meadow.Foundation/</PackageProjectUrl>
+    <PackageId>Meadow.Foundation.ICs.IOExpanders.Cp2112</PackageId>
+    <RepositoryUrl>https://github.com/WildernessLabs/Meadow.Foundation</RepositoryUrl>
+    <PackageTags>Meadow.Foundation,IOExpanders,expander,IO,CP2112</PackageTags>
+    <Version>0.1.0</Version>
+      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Description>CP2112 USB IOExpander for GPIO, I2C, SPI on Windows</Description>
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Cp2112/Driver/ICs.IOExpanders.Cp2112.csproj:17]: Add publish flag and set to false

File: Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ft232h/Driver/Ft232h.cs [hunk 4]
```diff
@@ -1,330 +1,139 @@
...
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading;
 
-namespace Meadow.Foundation.ICs.IOExpanders
+namespace Meadow.Foundation.ICs.IOExpanders;
+
+/// <summary>
+/// Represents a DS3502 digital potentiometer
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ft232h/Driver/Ft232h.cs:8]: Update XML comment

File: Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ft232h/Driver/Ft23xxI2cBus.cs [hunk 5]
```diff
@@ -0,0 +1,83 @@
+﻿using Meadow.Hardware;
+using System;
+using System.IO;
+
+namespace Meadow.Foundation.ICs.IOExpanders;
+
+public sealed class Ft23xxI2cBus : II2cBus, IDisposable
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ft232h/Driver/Ft23xxI2cBus.cs:7]: Add XML comments

File: Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ft232h/Samples/Ft232h_Windows_Sample/Ft232h_Windows_Sample.csproj [hunk 6]
```diff
@@ -14,6 +14,9 @@
   </ItemGroup>
 
   <ItemGroup>
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ft232h/Samples/Ft232h_Windows_Sample/Ft232h_Windows_Sample.csproj:16]: Should probably move this sample to another repo

File: Source/Meadow.Foundation.Peripherals/Sensors.Camera.Amg8833/Samples/Amg8833_RasPi_Sample/Amg8833_RasPi_Sample.csproj [hunk 7]
```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/Sensors.Camera.Amg8833/Samples/Amg8833_RasPi_Sample/Amg8833_RasPi_Sample.csproj:1]: Should move sample

File: Source/Meadow.Foundation.Peripherals/Sensors.Camera.Amg8833/Samples/Amg8833_Windows_Sample/Amg8833_Windows_Sample.csproj [hunk 8]
```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/Sensors.Camera.Amg8833/Samples/Amg8833_Windows_Sample/Amg8833_Windows_Sample.csproj:1]: Move sample

File: Source/Meadow.Foundation.Peripherals/Sensors.Camera.Amg8833/Driver/Amg8833.cs [hunk 9]
```diff
@@ -0,0 +1,136 @@
+﻿using Meadow.Hardware;
```
Comment (adrianstevens) [inline Source/Meadow.Foundation.Peripherals/Sensors.Camera.Amg8833/Driver/Amg8833.cs:1]: Needs standard sample

File: Source/Meadow.Foundation.Peripherals/Displays.AsciiConsole/Driver/AsciiConsoleDisplay.cs [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+    public AsciiConsoleDisplay(int width, int height)
+    {
+        Console.Clear();
+        Console.SetCursorPosition(0, 0);
+
+        _buffer = new CharacterBuffer(width, height);
+        PixelBuffer.Clear();
+    }
+
+    public void Clear(bool updateDisplay = false)
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Displays.AsciiConsole/Driver/AsciiConsoleDisplay.cs:29]: Shouldnt all these public methods have XML comments?

**Review by jorgedevs — APPROVED** [review]

**Pull request merged**